stages/org.osbuild.kickstart

```diff
@@ -122,6 +122,30 @@ SCHEMA = r"""
       }
     }
   },
+  "ostreecontainer": {
+    "type": "object",
+    "required": ["url"],
+    "additionalProperties": false,
+    "properties": {
+      "stateroot": {
+        "type": "string"
+      },
+      "url": {
+        "type": "string"
+      },
+      "transport": {
+        "type": "string",
+        "enum": ["registry", "oci", "oci-archive"]
+      },
```

Comment:
Should we mention the default here (`registry`) for our own reference so we don't need to refer to the docs when using the stage?  Though, that might imply that if left unset it will be used as `--transport registry`, which isn't the case.  It's probably fine without.